internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerApiConstants.java

```diff
@@ -8,6 +8,7 @@ public class ControllerApiConstants {
   public static final String BATCH_JOB_HEARTBEAT_ENABLED = "batch_job_heartbeat_enabled";
 
   public static final String NAME = "store_name";
+  public static final String STORE_PARTITION = "partition";
```

Comment:
can we call it `store_partition` so the value matches the variable name?

Reply:
Ok. Instead of using `VERSION`, I am adding `STORE_VERSION` to be consistent